version.py

```diff
@@ -29,7 +29,7 @@
 #
 # Changes should be accompanied by documentation explaining what the
 # new level means in templates/zerver/api/changelog.md.
-API_FEATURE_LEVEL = 33
+API_FEATURE_LEVEL = 34
```

Comment:
Need to bump this to 35 when you rebase.

Reply:
Done!